keystone_json_assignment/json.py

```diff
@@ -76,56 +78,68 @@ def _get_public_id(self, user_id):
             'local_id': user_id,
             'entity_type': 'user'})
 
+    def load_user_project_map(self):
+        mtime = datetime.datetime.utcfromtimestamp(
+            os.path.getmtime('/etc/keystone/user-project-map.json'))
+        if self.userprojectmap == {} or mtime > self.last_loaded:
+            with open('/etc/keystone/user-project-map.json', 'r') as f:
+                userprojectmap = yaml.load(f)
```

Comment:
does it make sense to wrap this in a `try...except` block ?

Reply:
You mean to catch issues if the file doesn't exist or is invalid yaml/json? We can do that but I think it's not related to the purpose of this PR, or do you mean something else?